src/app/components/team/SmoochBot/SmoochBotComponent.js

```diff
@@ -29,10 +29,10 @@ const SmoochBotComponent = ({
   const [currentLanguage, setCurrentLanguage] = React.useState(defaultLanguage);
   const languages = team.get_languages ? JSON.parse(team.get_languages) : [defaultLanguage];
 
-  const installation = team.team_bot_installations.edges.find(i => i.node.team_bot.name === 'Smooch');
-  const bot = installation ? installation.node.team_bot : null;
+  const installation = team.smooch_bot;
+  const bot = installation ? installation.team_bot : null;
```

Comment:
I think `installation` and `bot` variables are unnecessary now. `team.smooch_bot` will return `null` if not installed, so you can just refer to `team.smooch_bot` everywhere down the file